src/node.h

```diff
@@ -508,8 +508,6 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
 #define NODE_MODULE_CONTEXT_AWARE(modname, regfunc)                   \
   NODE_MODULE_CONTEXT_AWARE_X(modname, regfunc, NULL, 0)
 
-#define NODE_MODULE_CONTEXT_AWARE_BUILTIN(modname, regfunc)           \
-  NODE_MODULE_CONTEXT_AWARE_X(modname, regfunc, NULL, NM_F_BUILTIN)   \
```

Comment:
Probably a good idea but this is definitely `semver-major`.

Reply:
@Fishrock123 What here is semver-major?

I’m on the fence myself but *if* it is semver-major, then that is the case because embedders need to make an extra function call – not because of the other changes?